templates/UmbracoPackageRcl/.template.config/template.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://json.schemastore.org/template.json",
+  "author": "Umbraco HQ",
+  "classifications": [
+    "Web",
+    "CMS",
+    "Umbraco",
+    "Package",
+    "Plugin",
+    "Razor Class Library"
+  ],
+  "name": "Umbraco Package RCL",
+  "description": "An empty Umbraco package/plugin (Razor Class Library).",
+  "groupIdentity": "Umbraco.Templates.UmbracoPackageRcl",
+  "identity": "Umbraco.Templates.UmbracoPackageRcl.CSharp",
+  "shortName": "umbracopackage-rcl",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "sourceName": "UmbracoPackage",
+  "defaultName": "UmbracoPackage1",
+  "preferNameDirectory": true,
+  "symbols": {
+    "Framework": {
+      "displayName": "Framework",
+      "description": "The target framework for the project.",
+      "type": "parameter",
+      "datatype": "choice",
+      "choices": [
+        {
+          "displayName": ".NET 7.0",
+          "description": "Target net7.0",
+          "choice": "net7.0"
+        }
+      ],
+      "defaultValue": "net7.0",
+      "replaces": "net7.0"
+    },
+    "UmbracoVersion": {
+      "displayName": "Umbraco version",
+      "description": "The version of Umbraco.Cms to add as PackageReference.",
+      "type": "parameter",
+      "datatype": "string",
+      "defaultValue": "11.0.0",
```

Comment:
If this PR gets merged after https://github.com/umbraco/Umbraco-CMS/pull/13481, this value should also be updated to `*`...